h2o-py/tests/testdir_misc/pyunit_ice_plot.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import print_function
+
+import sys
+
+sys.path.insert(1,"../../")
+import matplotlib
+matplotlib.use("Agg")  # remove warning from python2 (missing TKinter)
+import h2o
+import matplotlib.pyplot
+from tests import pyunit_utils
+from h2o.estimators import *
+
+
+def test_original_values():
```

Comment:
This test would be actually a great example for test parametrization, as most of the logic is shared! I'm not sure what test framework you use, but e.g. in pytest, this is achieved using [`parametrize` decorator.](https://docs.pytest.org/en/6.2.x/parametrize.html). Consider if something like that is possible also in this codebase 🙂